appveyor.yml

```diff
@@ -31,10 +31,6 @@ environment:
     job_group: ps
     appveyor_build_worker_image: Visual Studio 2022
 
-  - job_name: PowerShell 6

```

Comment:
Not testing on PowerShell 6 anymore?

Reply:
IISAdministration, WebAdministration, and Microsoft.Web.Administration assembly don't work under PowerShell 6.